yesod-test/test/main.hs

```diff
@@ -329,7 +332,7 @@ main = hspec $ do
                     checkByLabel "Red"
                     checkByLabel "Gray"
                     addToken
-                bodyContains "colorCheckBoxes = [Gray,Red]"
+                bodyContains "colorCheckBoxes = [Red,Gray]" -- Changes for https://github.com/yesodweb/yesod/issues/1846 affects checkByLabel function, too. Reversed here from [Gray,Red]
```

Comment:
I don't think this comment is necessary.

Reply:
Ok, I will delete it.